eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1456,7 +1461,7 @@ protected function internalPublishVersion(APIVersionInfo $versionInfo, $publicat
 
         $metadataUpdateStruct = new SPIMetadataUpdateStruct();
         $metadataUpdateStruct->publicationDate = isset($publicationDate) ? $publicationDate : time();
```

Comment:
Both are fine with me, and second is indeed more readable so +1 from me on that.